docs/before_you_begin.rst

```diff
@@ -11,7 +11,7 @@ and sources, but no tool can guarantee safety. This guide will
 instruct you in installing and configuring SecureDrop, but it does not
 explain how to use it safely and effectively. Put another way: at the
 end of this guide, you will have built a car; you will not know how to
-drive. Make sure to review the :doc:`deployment_practices` to get the
+drive. Make sure to review the :ref:`Deployment` to get the
```

Comment:
It reads as **Make sure to review the Overview to get the most**

Reply:
![screen shot 2017-10-02 at 5 14 22 pm](https://user-images.githubusercontent.com/7832803/31104940-2f821edc-a795-11e7-90bf-e0f31d39d5a9.png)